packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -2399,15 +2405,20 @@
   "List_of_Direct_Messages": "List of Direct Messages",
   "Omnichannel": "Omnichannel",
   "Livechat": "Livechat",
+  "Livechat_abandoned_rooms_action": "How to handle Visitor Abandonment",
   "Livechat_abandoned_rooms_closed_custom_message": "Custom message when room is automatically closed by visitor inactivity",
   "Livechat_agents": "Omnichannel agents",
   "Livechat_Agents": "Agents",
+  "Livechat_allow_manual_on_hold": "Allow agents to manually place chat On Hold",
   "Livechat_AllowedDomainsList": "Livechat Allowed Domains",
   "Livechat_Appearance": "Livechat Appearance",
+  "Livechat_auto_close_on_hold_chats_custom_message": "Custom Message to be sent when a room in On-Hold queue gets automatically closed by the system",
+  "Livechat_auto_close_on_hold_chats_timeout": "How long the chat will remain in the On Hold queue until it's automatically closed by the system (in seconds) ?",
```

Comment:
I think the setting title should be shorter and straightforward and use the setting description to share more details on how it works.

Reply:
Okay, changed the title to `How long to wait before closing a chat in On Hold Queue ?`